app/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
     id("kotlin-parcelize")
+    id("androidx.navigation.safeargs.kotlin")
```

Comment:
half the plugins are `id '....'`, and half are `id("....")`

Let's be tidy with our code :)